src/js/stream.js

```diff
@@ -211,7 +212,7 @@ class Stream {
 
 		this.streamEl.parentNode.insertBefore(signedInMessage, this.streamEl);
 
-		document.querySelector('.o-comments__edit-display-name').onclick = () => {
+		document.querySelector(`#${editButtonId}`).onclick = () => {
```

Comment:
We could use the method `document.getElementById` instead of `document.querySelector` here, it takes a string which is the ID so we won't need to construct a new string like we have to for `document.querySelector`.

Reply:
This is done now. Thanks.